apps/omg/test/omg/typed_data_sign_test.exs

```diff
@@ -0,0 +1,209 @@
+# Copyright 2018 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.TypedDataSignTest do
+  @moduledoc false
+
+  use ExUnitFixtures
+  use ExUnit.Case, async: true
+
+  alias OMG.Crypto
+  alias OMG.State.Transaction
+  alias OMG.TypedDataSign
+  alias OMG.Utxo
+
+  require Utxo
+
+  @chain_id 4
+  @test_domain_separator TypedDataSign.Config.domain_separator(
+                           "OMG Network",
+                           "1",
+                           @chain_id,
+                           "1C56346CD2A2Bf3202F771f50d3D14a367B48070" |> Base.decode16!(case: :mixed),
+                           "f2d857f4a3edcb9b78b4d503bfe733db1e3f6cdc2b7971ee739626c97e86a558"
+                           |> Base.decode16!(case: :mixed)
+                         )
+
+  deffixture hardcoded_randoms() do
```

Comment:
you can comfortably live without `deffixture` in this case and have `inputs, outputs, metadata` as fields of the `context` resulting from `setup`.

We need to be extra wary of what happens when a bit later some change makes these tests red - since the test uses a lot of magic values, we should be super clear on how to treat the failing tests.